### DISCUSSION TITLE
Art Class v2 - Papa's Burgeria

### DISCUSSION THREAD

**Discussion by giscus** [proudparrot2/artclass-v2#1] [Game Comments]
# Art Class v2 - Papa's Burgeria

https://artclass.site/games/papas-burgeria.html

**Comment by proudparrot2** [comment]
Let us know what you think about the new comments feature :)

**Comment by pavlito1** [comment]
Great, I play this game Papa's Burgeria here [https://papas-games.com](https://papas-games.com)
